better error message if the maven tooling can't find the kubernetes master URL

see
```

[DEBUG] -- end configuration --
[INFO] Using kubernetes at: null in namespace default
[INFO] Kubernetes JSON: /Users/tim/Dev/quickstarts/quickstarts/war/camel-servlet/target/classes/kubernetes.json
Jul 14, 2015 11:12:29 AM org.apache.cxf.jaxrs.client.JAXRSClientFactoryBean createWebClient
SEVERE: java.lang.NullPointerException : null
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.967 s
[INFO] Finished at: 2015-07-14T11:12:29+02:00
[INFO] Final Memory: 23M/400M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.fabric8:fabric8-maven-plugin:2.2.7:apply (default-cli) on project quickstart-war-camel-servlet: HTTP 500 Internal Server Error: java.lang.NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:fabric8-maven-plugin:2.2.7:apply (default-cli) on project quickstart-war-camel-servlet: HTTP 500 Internal Server Error
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException: HTTP 500 Internal Server Error
	at io.fabric8.maven.ApplyMojo.execute(ApplyMojo.java:234)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 20 more
Caused by: javax.ws.rs.WebApplicationException: HTTP 500 Internal Server Error
	at io.fabric8.kubernetes.api.KubernetesClient.handle404ByReturningNull(KubernetesClient.java:1724)
	at io.fabric8.kubernetes.api.KubernetesClient.getTemplate(KubernetesClient.java:777)
	at io.fabric8.kubernetes.api.Controller.applyTemplate(Controller.java:287)
	at io.fabric8.maven.ApplyMojo.execute(ApplyMojo.java:177)
	... 22 more
Caused by: java.lang.RuntimeException: java.lang.NullPointerException
	at org.apache.cxf.jaxrs.client.JAXRSClientFactoryBean.createWebClient(JAXRSClientFactoryBean.java:233)
	at org.apache.cxf.jaxrs.client.WebClient.create(WebClient.java:177)
	at org.apache.cxf.jaxrs.client.WebClient.create(WebClient.java:137)
	at io.fabric8.kubernetes.api.KubernetesFactory.createWebClient(KubernetesFactory.java:274)
	at io.fabric8.kubernetes.api.KubernetesFactory.createWebClient(KubernetesFactory.java:265)
	at io.fabric8.kubernetes.api.KubernetesFactory.createWebClient(KubernetesFactory.java:260)
	at io.fabric8.kubernetes.api.KubernetesFactory.createKubernetesExtensions(KubernetesFactory.java:249)
	at io.fabric8.kubernetes.api.KubernetesClient.getKubernetesExtensions(KubernetesClient.java:116)
	at io.fabric8.kubernetes.api.KubernetesClient$10.call(KubernetesClient.java:780)
	at io.fabric8.kubernetes.api.KubernetesClient$10.call(KubernetesClient.java:777)
	at io.fabric8.kubernetes.api.KubernetesClient.handle404ByReturningNull(KubernetesClient.java:1716)
	... 25 more
Caused by: java.lang.NullPointerException
	at org.apache.cxf.jaxrs.JAXRSServiceImpl.getName(JAXRSServiceImpl.java:101)
	at org.apache.cxf.jaxrs.AbstractJAXRSFactoryBean.createEndpointInfo(AbstractJAXRSFactoryBean.java:149)
	at org.apache.cxf.jaxrs.AbstractJAXRSFactoryBean.createEndpoint(AbstractJAXRSFactoryBean.java:219)
	at org.apache.cxf.jaxrs.client.JAXRSClientFactoryBean.createWebClient(JAXRSClientFactoryBean.java:220)
	... 35 more
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/fabric8-maven-plugin/src/main/java/io/fabric8/maven/ApplyMojo.java
+++ b/fabric8-maven-plugin/src/main/java/io/fabric8/maven/ApplyMojo.java
@@ -155,6 +155,11 @@ public class ApplyMojo extends AbstractFabric8Mojo {
             }
         }
         KubernetesClient kubernetes = getKubernetes();
+        
+        if (kubernetes.getMasterUrl() == null) {
+        	throw new MojoFailureException("Can't find Kubernetes master URL");
+        }
+        
         getLog().info("Using kubernetes at: " + kubernetes.getMasterUrl() + " in namespace " + getNamespace());
 
         getLog().info("Kubernetes JSON: " + json);

